vllm/model_executor/model_loader/tensorizer.py

```diff
@@ -222,13 +233,6 @@ def add_cli_args(
             help="The endpoint for the S3 bucket. Can also be set via the "
             "S3_ENDPOINT_URL environment variable.",
         )
-        group.add_argument(
-            "--vllm-tensorized",
-            action="store_true",
-            help="If enabled, indicates that the serialized model is a vLLM "
-            "model. This is used to determine the behavior of the "
-            "TensorDeserializer when loading tensors from a "
-            "serialized model.")
```

Comment:
Should we bring this back and add a depreciation warning?

Reply:
We could, but with the refactor from #4097 tensorizer args can't be exposed from the CLI directly anymore anyway.  They can only be encoded in `--model-loader-extra-config`